docs/topics/whatsnew18.md

```diff
@@ -223,6 +223,53 @@ Here are some of the key differences between the two layouts:
 | main  | src/main/AndroidManifest.xml  | src/<b>android</b>Main/AndroidManifest.xml  |
 | debug | src/debug/AndroidManifest.xml | src/<b>android</b>Debug/AndroidManifest.xml |
 
+#### The relation between Android tests and common tests
+
+The new Android source set layout changes the relation between Android-instrumented tests (renamed to `androidInstrumentedTest` in the new layout)
+and common tests.
+
+Previously, there was a default `dependsOn` relation between `androidAndroidTest` and `commonTest`. In practice, it meant the following:
+
+* The code in `commonTest` was available in `androidAndroidTest`.
+* `expect` declarations in `commonTest` had to have corresponding `actual` realizations in `androidAndroidTest`.
+* Tests declared in `commonTest` were running as Android instrumented tests.
```

Comment:
Minor: It reads to me personally as if they were running _only_ as Android instrumented tests, which is not correct. Maybe a rephrasing would help?

`Tests declared in commonTest were participating in Android instrumented test runs`

`Tests declared in commonTest were also running as Android instrumented tests`

`During Android instrumented tests run, all tests declared in commonTest were launched as well`